pkg/pac/pac.go

```diff
@@ -468,6 +469,13 @@ func ConfigurePreviewChanges(client *gitlab.Client, projectID int) string {
 	return pipelineName
 }
 
+func AssertPACInfoInstall() {
+	_, err := opc.GetOpcPacInfoInstall()
+	if err != nil {
+		testsuit.T.Fail(fmt.Errorf("failed to get pac info: %v", err))
+	}
```

Comment:
You only check that the output is not empty. IMHO we could assert the actual PAC version (from env variable `PAC_VERSION`) and namespace (always `openshift-pipelines`) here.